rust_src/src/buffers.rs

```diff
@@ -91,6 +96,17 @@ impl LispBufferRef {
     pub fn is_live(self) -> bool {
         LispObject::from_raw(self.name).is_not_nil()
     }
+
+    #[inline]
+    pub fn fetch_byte(&self, n: ptrdiff_t) -> u8 {
+        let base_addr = if n >= self.gpt_byte() {
```

Comment:
I'd prefer we called this `offset` -- it's not actually an address (0 wouldn't make sense as an address here).